testsuite/forge/src/backend/k8s/macros.rs

```diff
@@ -0,0 +1,32 @@
+#[macro_export]
+macro_rules! gvk {
+    ($kind:expr) => {{
+        match $kind {
+            k8s_openapi::api::core::v1::Pod::KIND => Some(kube::api::GroupVersionKind {
```

Comment:
This is annoying but gives us a nice alias / interface where we can say "Pod" instead the version etc

You could also construct these manually or make funciton for them and always pass it but this lets it be dynamic from the resource passed